eventemails.php

```diff
@@ -0,0 +1,28 @@
+<?php
+header("Content-Type: text/plain");
+require_once ("../connectsodb.php");
+require_once  ("checksession.php"); //Check to make sure user is logged in and has privileges
+require_once  ("functions.php");
+userCheckPrivilege(2);
+
+$event = intval($_POST['myID']);
+//semester teams tournament hardcoded, change later
+$emails = "";
+
+$query = "SELECT * FROM `tournamentevent` INNER JOIN `teammateplace` ON `tournamentevent`.`tournamenteventID` = `teammateplace`.`tournamenteventID` INNER JOIN `student` ON `teammateplace`.`studentID` = `student`.`studentID` WHERE `tournamentID` = 12 AND eventID = $event";
```

Comment:
This event emails is specific for a tournamentID = 12.  This will work for now, but it will break in the future.  Shouldn't this instead be enabled for all tournaments?

Reply:
I changed it to include all tournaments after july 31st of the current scioly year - 1, so as to include the fall roster and all tournaments that come after that.